optimum/bettertransformer/models/encoder_models.py

```diff
@@ -232,6 +232,53 @@ def forward(self, hidden_states, attention_mask, *_):
             hidden_states = hidden_states.to_padded_tensor(0.0)
         return (hidden_states,)
 
+    def _replace_to_original_module(self):
```

Comment:
```suggestion
    def  _revert_back_to_original_module(self):
```

Reply:
fixed in [8ee8df5](https://github.com/huggingface/optimum/pull/676/commits/8ee8df53d6685110ca2db78b94e9b0c0f49d533e)